Bottom: 0ae559eafebd55df0312dedb20512b51ede17d42
Top:    694129b6859c6228c5aa7cfa347afbed42e3c7a3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 17:11:42 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
